examples/11-filtering-handlers/example.py

```diff
@@ -2,33 +2,43 @@
 
 
 @kopf.on.create('zalando.org', 'v1', 'kopfexamples', labels={'somelabel': 'somevalue'})
-def create_with_labels_satisfied(logger, **kwargs):
-    logger.info("Label satisfied.")
+def create_with_labels_matching(logger, **kwargs):
+    logger.info("Label is matching.")
 
 
-@kopf.on.create('zalando.org', 'v1', 'kopfexamples', labels={'somelabel': None})
-def create_with_labels_exist(logger, **kwargs):
-    logger.info("Label exists.")
+@kopf.on.create('zalando.org', 'v1', 'kopfexamples', labels={'somelabel': kopf.PRESENT})
+def create_with_labels_present(logger, **kwargs):
+    logger.info("Label is present.")
+
+
+@kopf.on.create('zalando.org', 'v1', 'kopfexamples', labels={'unexistent': kopf.ABSENT})
+def create_with_labels_absent(logger, **kwargs):
+    logger.info("Label is absent.")
 
 
 @kopf.on.create('zalando.org', 'v1', 'kopfexamples', labels={'somelabel': 'othervalue'})
-def create_with_labels_not_satisfied(logger, **kwargs):
-    logger.info("Label not satisfied.")
+def create_with_labels_mismatch(logger, **kwargs):
```

Comment:
Right.

The examples are used as the functional tests. But we also have unit-test for labels/annotations matching & mismatching anyway. So, this precise level of testing is not needed. It can be reduced to pure demonstration of the markers and syntax only.

Removed.